src/sentry/api/issue_search.py

```diff
@@ -192,7 +189,7 @@ def convert_category_value(
         group_category = getattr(GroupCategory, category.upper(), None)
         if not group_category:
             raise InvalidSearchQuery(f"Invalid category value of '{category}'")
-        results.extend(get_group_types_by_category(group_category.value))
+        results.extend(GROUP_TYPE_REGISTRY.get_by_category(group_category.value))
```

Comment:
Just changing to use the registry directly, [as stated in this todo comment](https://github.com/getsentry/sentry/blob/af250470a4951f2291b6fc0063eb2e899c4d59b3/src/sentry/issues/grouptype.py#L277)